docs/_snippets/features/table-styling.html

```diff
@@ -1,54 +1,76 @@
 <div id="snippet-table-styling">
-	<p>The ultimate table of <a href="https://en.wikipedia.org/wiki/List_of_coffee_drinks">coffee drinks</a>:</p>
-	<table>
-		<thead>
-			<tr>
-				<th rowspan="2"></th>
-				<th scope="col" rowspan="2">Espresso</th>
-				<th scope="colgroup" colspan="2">Milk</th>
-				<th scope="col" rowspan="2">Water</th>
-			</tr>
-			<tr>
-				<th scope="col">Steamed</th>
-				<th scope="col">Foam</th>
-			</tr>
-		</thead>
-		<tbody>
-			<tr>
-				<th scope="row">Cappuccino</th>
-				<td>yes</td>
-				<td>yes</td>
-				<td>yes</td>
-				<td>no</td>
-			</tr>
-			<tr>
-				<th scope="row">Macchiato</th>
-				<td>yes</td>
-				<td>no</td>
-				<td>yes</td>
-				<td>no</td>
-			</tr>
-			<tr>
-				<th scope="row">Latte</th>
-				<td>yes</td>
-				<td>yes</td>
-				<td>no</td>
-				<td>no</td>
-			</tr>
-			<tr>
-				<th scope="row">Americano</th>
-				<td>yes</td>
-				<td>no</td>
-				<td>no</td>
-				<td>yes</td>
-			</tr>
-		</tbody>
-	</table>
+	<h3>Balance Sheet &amp; Cash Flow Statement Information</h3>
+	<figure class="table">
+		<table style="border-bottom:1px solid;border-left:1px solid;border-right:1px solid;border-top:1px solid;">
+			<thead>
+				<tr>
+					<th style="background-color:#eee;width:317px;" colspan="1" rowspan="2"><p>&nbsp;</p><p>&nbsp;</p></th>
+					<th style="background-color:#1d7373;text-align:center;width:224px;" colspan="3">Full Year (unaudited)</th>
+					<th style="background-color:#1d7373;text-align:center;width:360px;" colspan="4">2013 (unaudited)</th>
+				</tr>
+				<tr>
+					<th style="background-color:#5ccccc;width:78px;">2011</th>
```

Comment:
Shouldn't we use colors that are available in out colorpicker? So e.g. someone may reuse the same color?